src/content/1.7/scale/optimizations.md

```diff
@@ -260,3 +260,35 @@ if (!defined('_PS_DEBUG_PROFILING_')) {
     define('_PS_DEBUG_PROFILING_', false);
 }
 ```
+
+### 8) PrestaShop database cleaning
+
+Some SQL tables of PrestaShop can grow very big in time. Some of the data being stored there is sometimes not relevant anymore or not needed anymore.
+
+Some notable examples:
+
+#### Logs table
+
+You can regularly clean your _log_ SQL table from old logs.
```

Comment:
@matks we should say `_log_ SQL table` or` _log_ database table` ?

Reply:
I thought that "SQL table" is more common so it's easier to understand than "database table". Both are fine to me, I'm just looking for a statement that most people will get

Reply:
```suggestion
You can regularly clean your _log_ database table from old logs.
```
or
```suggestion
You can regularly clean your _log_ MySQL table from old logs.
```

SQL is not relevant : it's a programmation language.